presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/SemiTransactionalHiveMetastore.java

```diff
@@ -1014,7 +1033,7 @@ public synchronized ConnectorCommitHandle commit()
         try {
             switch (state) {
                 case EMPTY:
-                    return EMPTY_HIVE_COMMIT_HANDLE;
+                    return new HiveCommitHandle(lastDataCommitTimes, ImmutableMap.of());
```

Comment:
I guess my real question here is what does it mean if the state is EMPTY?  I assumed it was something like nothing to commit, so in that case ow can it end up with lastDataCommitTimes that need to be passed along?  But maybe it means something else.